src/ml/neural_net/mps_layer_instance_norm_data_loader.h

```diff
@@ -0,0 +1,92 @@
+#ifndef mps_layer_instance_norm_data_loader_h
+#define mps_layer_instance_norm_data_loader_h
+
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h> 
+
+typedef struct{
+  int size;
+  float beta1;
+  float beta2;
+  float epsilon;
+  float lr_t;
+} params_t;
+
+API_AVAILABLE(macos(10.14))
+@interface InstanceNormDataLoader: NSObject <MPSCNNInstanceNormalizationDataSource> {
+  NSString *mName;
```

Comment:
These instance variables are largely implementation details. Let's move them to the .mm file. In addition, we use properties and let ObjC auto-synthesize the instance variables themselves.

Look at the code in https://github.com/apple/turicreate/blob/master/src/ml/neural_net/TCMPSImageAugmenting.h for an example of this pattern